ECMA2Yaml/ECMAHelper/Models/ReflectionItem.cs

```diff
@@ -25,7 +25,8 @@ public enum ItemType
         Operator,
         Container,
         AttachedEvent,
-        AttachedProperty
+        AttachedProperty,
+        EnumClass
```

Comment:
instead of introducing another item type, I'm thinking maybe we can catch this case in ECMAStore.Build() and clone the type object then.